packages/dx-react-scheduler-material-ui/src/templates/appointment-tooltip/layout.jsx

```diff
@@ -3,9 +3,11 @@ import * as PropTypes from 'prop-types';
 import Popover from '@material-ui/core/Popover';
 import AccessTime from '@material-ui/icons/AccessTime';
 import Grid from '@material-ui/core/Grid';
-import moment from 'moment';
 import { withStyles } from '@material-ui/core/styles';
 
+const hourMinuteOptions = { hour: 'numeric', minute: 'numeric' };
```

Comment:
This constant and other similar are duplicated in several files. Looks like they all should be declared in some single place (like *date-navigator/helpers.ts*) and exported in other places, shouldn't they?